src/Sylius/Bundle/ResourceBundle/AbstractResourceBundle.php

```diff
@@ -147,4 +147,12 @@ protected function getConfigFilesPath()
             strtolower($this->getDoctrineMappingDirectory())
         );
     }
+
+    /**
+     * @return string
+     */
+    protected function getObjectManagerParameter()
+    {
+      return  sprintf('%s.object_manager', $this->getBundlePrefix());
```

Comment:
@videni Could you please fix that little CS and we are good to merge, thanks! :)